openedx/openedx-platform
# Registration form: an optional "Confirm password" field

Continuation of the pull request https://github.com/edx/edx-platform/pull/12226

**Background:** Users often misspell their passwords and can't login a 2d time

**Studio Updates:** None.

**LMS Updates:** an optional "Confirm password" can be shown in the registration form. If the field is present a copy of password must be entered in it.

**Manual test instructions:**
Change lms.env.json:
`"REGISTRATION_EXTRA_FIELDS": {
        "password_copy": "required", 
`
Open http://127.0.0.1:8000/register and try to enter different "Password" and "Confirm password" and see a validation message.
